### DISCUSSION TITLE
Publishing created package

### DISCUSSION THREAD

**Discussion by Leengit** [Project-MONAI/monai-deploy-app-sdk#347] [Q&A] [answered]
I seek advice on how I can push the created package to somewhere accessible to the public.  I use a typical command of the form
```shell
monai-deploy \
    package my_depoly_app.py \
    --requirement my_requirements.txt \
    --tag my_app:latest \
    --model my_model.zip
```
and I see the docker image when I type `docker image ls`.  I would  like to make it publicly retrievable via a `docker pull` command, but when I tag the image and `docker push` to `ghcr.io` using my newly created personal access token, it silently fails; I think the 18 GB size of the image is too big.  I might instead be able to push a `Dockerfile` to a relevant GitHub repository but I don't have the Dockerfile that creates the base image that monai-depoly uses, nor do I know how I should then add the TorchScript model.  And I worry that, even succeeding with those steps, GitHub will complain about the size of the image that it creates by following the Dockerfile to do the docker build.

In short, I'm short on ideas.  Best practices on how to make this large docker image available via a `docker pull` command would be much appreciated!  Thank you.

**Comment by MMelQin** [comment, marked as answer]
Thanks @Leengit for the question, and sorry for this belated reply (I was doing something that is exactly the same as you were trying to)

I actually have successfully pushed the MAPs, Lung Seg and Liver and Tumor Seg, to GitHub Container Registry, of course in the humongous image (sorry about it, but it is coming from the NV PyTorch base). I also published a ["Show and tell" topic](https://github.com/Project-MONAI/monai-deploy-app-sdk/discussions/340) on how to pull and run those two MAPs with a shell script.
